gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Nov 17 11:57:30 EST 2016
+#Mon Oct 30 13:59:18 CDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.9-all.zip
```

Comment:
There's no reason to change this just because IDEA suggested it. The `all` distribution includes gradle source that is not needed for a project like this. The binary distribution is completely adequate.

Reply:
On this note, I'd prefer if all of the gradle stuff is left untouched. It's an absolute pain to get to work, and changing it usually ends up breaking things.

Reply:
Ok, I see your points on this. Reverting to bin in next commit in a bit